tests/cli/helpers.go

```diff
@@ -181,6 +195,13 @@ func UnmarshalStdTx(t require.TestingT, c *codec.Codec, s string) (stdTx auth.St
 	return
 }
 
+func queryEvents(events []string) (out string) {
```

Comment:
Can we give this function a better name please? Something like `buildEventsQueryString`?